pkg/test/utils/subset.go

```diff
@@ -63,6 +64,19 @@ func IsSubset(expected, actual interface{}) error {
 		iter := reflect.ValueOf(expected).MapRange()
 
 		for iter.Next() {
+			key := iter.Key().String()
+			if strings.HasPrefix(key, "$REQUIRE_MISSING_") {
```

Comment:
Instead of adding this feature to the test harness, maybe we could implement the `-errors` file per the test harness spec? https://github.com/kudobuilder/kudo/blob/master/keps/0008-operator-testing.md#test-suite-directory-structure

Reply:
Good point, didn't know about that -errors feature. I'll try that on monday.

Reply:
Ha, I actually thought the `-errors` feature hadn't been implemented yet and was suggesting you implement that instead of this, but it looks like @nfnt implemented it already. Thanks @nfnt!